bin/build-cluster-catalog.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+"""Build and write out the NGC-star-clusters.fits catalog.
+
+"""
+import os
+import numpy as np
+import numpy.ma as ma
+from astropy.io import ascii
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'ascii'](https://app.codacy.com/manual/dstndstn/legacypipe/pullRequest?prid=4930373)